source/appModules/totalcmd64.py

```diff
@@ -1,2 +1,3 @@
 #Use the app module for Total Commander 32 bit version.

```

Comment:
```suggestion
# Use the app module for Total Commander 32 bit version.

```

Reply:
Same as above - this is not introduced in this PR and there is no point in modifying alias app Modules.